vmware-tanzu/tanzu-framework
# Add cli plugin for airgapped image copy

Signed-off-by: Vandana Pathak <pathakv@sjc-ubu-eng-127.vmware.com>

### What this PR does / why we need it
Add custom image repository pre-loading command to Tanzu CLI to replace bash script hack for users in airgapped environments
CLI Plugin  - **isolated-cluster**

Commands - 
- **download-bundle**   - Download images/bundle into local disk as TAR.  
- **upload-bundle** - Upload images to private repository

Ex : 
```sh

tanzu isolated-cluster download-bundle --source-repo --tkg-version --source-ca-certificate/--source-insecure

tanzu isolated-cluster upload-bundle --source-directory --destination-repo --destination-ca-certificate/--destination-insecure  
```




### Which issue(s) this PR fixes


Fixes # [4093](https://github.com/vmware-tanzu/tanzu-framework/issues/4093)

### Describe testing done for PR

- Run  tanzu isolated-cluster download-bundle with version v1.6.0 , its downloading all the tar files and yaml as expected.(with insure mode)
- Run tanzu isolated-cluster upload-bundle with destination repo Azure ACR , its uploading all the images to acr. 



### Release note

```release-note
CLI Plugin  - isolated-cluster

Commands - 
- download-bundle   - Download images/bundle into local disk as TAR.  
- upload-bundle - Upload images to private repository

Complete Command :
 
tanzu isolated-cluster download-bundle --source-repo --tkg-version --source-ca-certificate/--source-insecure

tanzu isolated-cluster upload-bundle --source-directory --destination-repo --destination-ca-certificate/--destination-insecure  
```



### Additional information

#### Special notes for your reviewer
